lib/codecov.js

```diff
@@ -238,11 +316,20 @@ var upload = function(args, on_success, on_failure){
   }
 
   // Detect .bowerrc
-  var bowerrc = execSync('cd '+root+' && test -f .bowerrc && cat .bowerrc || echo ""').toString().trim(), more_patterns = '';
+  var bowerrc;
+  if(!isWindows) {
+    bowerrc = execSync('cd '+root+' && test -f .bowerrc && cat .bowerrc || echo ""').toString().trim(), more_patterns = '';
+  } else {
+    bowerrc = execSync('cd '+root+' && if exist .bowerrc type .bowerrc').toString().trim(), more_patterns = '';
```

Comment:
Move `, more_patterns` to line 319 since it's meant to be defining it. same goes with the previous section.